qemu/qmp.go

```diff
@@ -993,7 +993,7 @@ func (q *QMP) ExecuteNetdevAddByFds(ctx context.Context, netdevType, netdevID st
 	}
 	if len(vhostFdNames) > 0 {
 		vhostFdNameStr := strings.Join(vhostFdNames, ":")
-		args["vhost"] = "on"
+		args["vhost"] = true
```

Comment:
I haven't managed to grind through all the layers of qemu/qmp yet, but I'm not sure this is correct: if you look at the file below, `vhost=[on|off]` (as are all the bool options fwics):

https://github.com/qemu/qemu/blob/master/qemu-options.hx#L2338

Reply:
I blame this commit https://github.com/qemu/qemu/commit/db2a380c84574d8c76d7193b8af8535234fe5156 , It changed the way to parse the `netdev_add` command, I wonder if there is a patch to fix this regression ... ?

Reply:
Yes, there seems that no change in the command line itself was done in the last few years. However, there's this regression pointed by @devimc, which was caused by "-netdev" switching to a new CLI parsing.

@ebblake, you mind adding your opinion here? Is this something that'll be fixed in the next release? Or is this a change that came to stay?

Reply:
Ok, seems it came to stay as re-adding such options would be not exactly aligned with the effort to sanitize QAPI / QMP interface.

@devimc, you'll have to do a check on the qemu version ... if you want to keep backwards compat.

In any case, we should be using libvirt in the future, as libvirt would be dealing with such changes.

Reply:
> we should be using libvirt in the future

ummm you know something that I don't ... tell me more :smile:

Reply:
> > we should be using libvirt in the future
> 
> ummm you know something that I don't ... tell me more

Well, I was thinking about https://github.com/kata-containers/runtime/issues/2227, but that's a different thing.